runtime/common/src/lib.rs

```diff
@@ -61,7 +64,7 @@ mod constants {
 	/// slot_duration()`.
 	///
 	/// Change this to adjust the block time.
-	pub const MILLISECS_PER_BLOCK: u64 = 6000;
+	pub const MILLISECS_PER_BLOCK: u64 = 12000;
```

Comment:
interesting. Are we sure?

Reply:
We can revisit this later but for now there is no way that you can produce a block in 6 seconds, it takes 12 seconds for any parachain.

Reply:
I reverted to 6 sec as my local network works well with this block time. In any case this is controlled by the relay chain and Cumulus ignores any number we put here.